### PR TITLE
fix(core): token/figma alignment

### PR DESCRIPTION
**Pull request by coryrylan**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the new behavior?
This is a first pass in fixing up minor style fixes to align the new core figma library and the core web components. There will be a follow up PR soon for the last handful of components.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/core/build/tokens.ts [hunk 1]
```diff
@@ -472,6 +472,7 @@ const aliases = {
     },
     interaction: {
       outline: token('Highlight solid 2px'),
+      outlineWebkit: token('5px auto -webkit-focus-ring-color'), // https://coryrylan.com/blog/dont-override-css-outline-focus-styles
```
Comment (coryrylan) [inline packages/core/build/tokens.ts:475]: Was missing this in a couple of places which broke the native contrast detection for webkit outline. Made it a token to make sure its consistent. The px value has to be used to trigger chromium/webkit to use its special outline contrast feature.

File: packages/core/src/styles/tokens/tokens.stories.ts [hunk 2]
```diff
@@ -397,8 +397,6 @@ export function interactionMenuDemo() {
       .cds-menu-mock {
```
Comment (coryrylan) [inline packages/core/src/styles/tokens/tokens.stories.ts:397]: Most of these changes in the file are to better represent how the interaction tokens will be used in components and help narrow down consistency between them. As more of these are implemented the demos will be replaced with the `cds-*` element.

File: packages/core/src/alert/alert.element.scss [hunk 3]
```diff
@@ -109,6 +105,7 @@ cds-internal-close-button {
   --fill-color: var(--icon-color);
 }
 
+// need?
```
Comment (hippee-lee) [inline packages/core/src/alert/alert.element.scss]: remove?

**Review by hippee-lee — APPROVED** [review]

**Review by bdryanovski — APPROVED** [review]

File: packages/core/src/navigation/navigation-start.element.ts [hunk 4]
```diff
@@ -18,7 +18,6 @@ import {
   state,
 } from '@cds/core/internal';
 import itemStyles from './navigation-item.element.scss';
-import styles from './navigation-start.element.scss';
```
Comment (coryrylan) [inline packages/core/src/navigation/navigation-start.element.ts:21]: Removed file, since it was empty it was logging unnecessary warnings out from rollup.

**Pull request merged**